CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # App Center SDK for Unity Change Log
 
+## Development version
+
+### App Center Crashes
+
+#### iOS
+
+* **[Fix]** Fix that 'Crashes' could not be started because of 'StartCrashes' method was stripped for a high managed stripping level.
```

Comment:
```suggestion
* **[Fix]** Fix that Crashes could not be started because of `StartCrashes` method was stripped for a high managed stripping level.
```